routes/data.js

```diff
@@ -189,10 +189,13 @@ exports.organizationsGet = async function (req, res) {
 };
 
 exports.organizationsPost = async function (req, res) {
-  const updatedOrganizationData = req.body;
+  const newOrgData = req.body;
+  newOrgData.acceptsDropOff = newOrgData.acceptsDropOff ? true : false;
+  newOrgData.acceptsPickUp = newOrgData.acceptsPickUp ? true : false;
+  newOrgData.acceptsShipping = newOrgData.acceptsShipping ? true : false;
```

Comment:
What are we doing here, casting to boolean?

Reply:
HTML checkboxes don't show up in the `POST` body unless they are checked. This allows us to update the checkbox values in the database if a checkbox becomes unchecked.

Reply:
Sorry, I'm still having trouble understanding. How does

``` newOrgData.acceptsDropOff = newOrgData.acceptsDropOff ? true : false;```

mutate the value of ```newOrgData.acceptsDropOff```?

Reply:
If the checkbox `acceptsDropOff` is not checked in the HTML body, it will not be posted. Therefore the only way I thought of telling the database to set its value to false was to check if `newOrgData.acceptsDropOff`'s value has not been posted, and then assign it to be `false`.

Reply:
Got it, I think you can replace this with ```!!newOrgData.acceptsDropOff```.